src/pivotal-ui/components/dropdowns/dropdowns.scss

```diff
@@ -159,7 +159,7 @@ var Dropdown = require('pui-react-dropdowns').Dropdown;
 This is the basic bootstrap dropdown:
 
 ```react_example_table
-<UI.Dropdown title='DropDown'>
+<UI.Dropdown title='DropDown' buttonClassName="my-btn-class">
```

Comment:
Turns out we have an example of using `buttonClassName` lower in the documentation. So we should just remove this hanging `buttonClassName`, and then this should be ready for merge.